### DISCUSSION TITLE
QuestPdf Blazor Server

### DISCUSSION THREAD

**Discussion by dtramzter** [QuestPDF/QuestPDF#1045] [Q&A] [answered]
Hello

I am making an application with Blazor Server and I am using QuestPdf and it works fine when I use it on the computer where Blazor is installed, but when I manage the program on another computer and create a new PDF, the PDF is displayed on the same computer where the Blazor server is started and it is not visible on the remote (local) computer. Why is this happening? How can I make the PDF visible from another computer?

**Comment by girlpunk** [comment, marked as answer]
This is likely expected behaviour. Starting a process will always result in it starting on the same computer. You will need to send the file to the browser to download and save to the computer running the browser, from which the user can then open the file.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
